test/components/navbar/user-actions-menu.spec.coffee

```diff
@@ -45,12 +41,12 @@ testWithRole = (roleType) ->
         courseModel.is_concept_coach = false
         CourseActions.loaded(courseModel, courseModel.id)
 
-      it 'should not have disallowed menus', (done) ->
+      it 'should not have disallowed menus', ->
```

Comment:
I don't think these are async and need the done?  but i can add them back in just in case